### ISSUE TITLE
[Features] Enable Assist in the Dashboard

### ISSUE THREAD

**Issue by lbouriez** [dwainscheeren/dwains-lovelace-dashboard#674]
Hello Dwains,
Since the HA toolbar is hidden in the dashboard, we cannot try Assist. Is there a way to disply the icon ?

**Comment by dwainscheeren** [comment]
I don't understand your question. What is Assist?

**Comment by aviadlevy** [comment]
> I don't understand your question. What is Assist?

https://www.home-assistant.io/blog/2023/02/01/release-20232/#meet-assist

**Comment by iroQuai** [comment]
That would be wonderful!

**Comment by thundergreen** [comment]
Yes.. opt-in for assist would be great.. maybe we can get it as a popup.

**Comment by dwainscheeren** [comment]
Thanks for reporting this.

Dwains Dashboard 3.9.0 has now been released, and the source files for DD3 are available in this repository.

DD3 is no longer under active feature development and will slowly be phased out. Active development continues in Dwains Dashboard Next:
https://github.com/dwainscheeren/dwains-dashboard-next

At the time of writing, Dwains Dashboard Next is still in alpha.

If this issue is a feature request, please check whether it fits Dwains Dashboard Next instead.

If this issue is a bug, please retest with DD3 version 3.9.0. If the bug is still reproducible there, please open a new issue with:

- Home Assistant version
- Dwains Dashboard version
- browser and device
- browser console errors
- clear reproduction steps

For more context, see #906.

For that reason, this older issue is being closed as part of the DD3 issue tracker cleanup.
